Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -1379,6 +1379,16 @@ theorem isOpen_analyticAt : IsOpen { x | AnalyticAt 𝕜 f x } := by
   exact mem_of_superset (EMetric.ball_mem_nhds _ hr.r_pos) fun y hy => hr.analyticAt_of_mem hy
 #align is_open_analytic_at isOpen_analyticAt
 
+variable {𝕜}
+
+theorem AnalyticAt.eventuallyAnalyticAt {f : E → F} {x : E} (h : AnalyticAt 𝕜 f x) :
+    ∀ᶠ y in 𝓝 x, AnalyticAt 𝕜 f y :=
+(isOpen_analyticAt 𝕜 f).mem_nhds h
+
+theorem AnalyticAt.existsMemNhdsAnalyticOn {f : E → F} {x : E} (h : AnalyticAt 𝕜 f x) :
```

Comment:
I think the naming convention is wrong. I would say `AnalyticAt.exists_mem_nhds_analyticOn`, but I'm not sure. The way I understand it is: separate different mathematical concepts with underscores, and start each mathematical concept with a lowercase letter.

You should ask on zulip, then we would both learn the right answer :-)

(Same thing for the previous lemma, my guess is that it should be `AnalyticAt.eventually_analyticAt`)